pkg/client/googlegroup_github/mapper.go

```diff
@@ -19,15 +19,80 @@ import (
 	"context"
 	"fmt"
 	"os"
+	"slices"
 	"strings"
 
 	"google.golang.org/protobuf/encoding/prototext"
 
 	"github.com/abcxyz/pkg/logging"
 	"github.com/abcxyz/team-link/apis/v1alpha3"
+	"github.com/abcxyz/team-link/pkg/github"
 	"github.com/abcxyz/team-link/pkg/groupsync"
 )
 
+// GroupMapper implements groupsync.OneToManyGroupMapper
+// For now it was be used as Mapper between GoogleGroup
+// and GitHubTeam.
+// Both GoogleGroupMapper and GitHubMapper have similar
+// variables and functions, this way we don't need to repeatly
+// write similar function for both on them.
+type GroupMapper struct {
+	mappings map[string][]string
+}
+
+func (m *GroupMapper) AllGroupIDs(ctx context.Context) ([]string, error) {
+	res := make([]string, 0, len(m.mappings))
+	for key := range m.mappings {
+		res = append(res, key)
+	}
+	slices.Sort(res)
+	return res, nil
+}
+
+func (m *GroupMapper) ContainsGroupID(ctx context.Context, key string) (bool, error) {
+	_, ok := m.mappings[key]
+	if !ok {
+		return false, nil
+	}
+	return ok, nil
```

Comment:
Hint: you can make this even simpler now. What happens if you remove L54-56?

Reply:
ahhhhhh, true, barin is slow today, the ok was for wrapping the error and now we don't need it.
Thank you!